Tasks/Common/nuget-task-common/externals.json

```diff
@@ -0,0 +1,19 @@
+{
+    "archivePackages": [
+        {
+            "archiveName": "NuGet.exe",
+            "url": "https://vstsagenttools.blob.core.windows.net/tools/NuGet/3.3.0/nuget.zip",
```

Comment:
Only be used at build time.  If we've used the wrong construct for that let us know, but we've validated the exes come down with the task and those are what we use (no reaching out during build).  @chrisrpatterson we were told all things pulled into the build use your blob container so that is what we've done here for the build.